adminSiteClient/PostsIndexPage.tsx

```diff
@@ -112,13 +135,29 @@ class PostRow extends React.Component<PostRowProps> {
                         to={`gdocs/${post.gdocSuccessorId}/preview`}
                         className="btn btn-primary"
                     >
-                        Preview
+                        <>
+                            <FontAwesomeIcon icon={faEye} /> Preview
+                        </>
                     </Link>
                     <button
                         onClick={async () => await this.onRecreateGdoc()}
                         className="btn btn-primary alert-danger"
                     >
-                        Recreate
+                        <FontAwesomeIcon
+                            icon={faRecycle}
+                            title={
+                                "Recreate the Gdoc, replacing existing content"
+                            }
+                        />
+                    </button>
+                    <button
+                        onClick={async () => await this.onUnlinkGdoc()}
```

Comment:
`onClick={this.onUnlinkGdoc}` is fine here. TS only complains if the method takes an argument 

I don't mind either way, just pointing it out in case you didn't know 🙂

Reply:
Yeah it reads a bit nicer :)